rules/windows/process_creation/proc_creation_win_renamed_pressanykey.yml

```diff
@@ -0,0 +1,28 @@
+title: Visual Studio NodejsTools PressAnyKey Renamed Execution
```

Comment:
like now the rule doesn't detect only renamed binaries but in different paths. If a correctly named binary is executed from a different location the rule claims something false. 
Either we say more genericly "suspicious execution" or say "renamed execution or from uncommon location". Which is a long title :see_no_evil:

Reply:
To keep it in the same "renamed" spirit, I will remove the path and leave only the image. And if it's executed from other paths with same name it'll still trigger the other related rule.